web/app/tournament/tourny_helpers.py

```diff
@@ -0,0 +1,79 @@
+from flask import current_app, render_template, redirect, url_for
+from flask import jsonify, request, Response, flash
+from app.models import db, Tournament, TournamentPlayer, Match, Round
+
+def empty_field(form_data, fetched_db_data):
+    """Treating an empty string returned by form as equal to None in DB"""
```

Comment:
Each doc string should read as 

(tacitly: this method) does blah

Ex in this case

Returns true if the form data or db data is empty.

I'd recommend naming like

is_empty_field(...):

https://www.python.org/dev/peps/pep-0257/